lib/src/main/java/nl/knaw/dans/lib/dataverse/AbstractApi.java

```diff
@@ -27,8 +27,9 @@ protected AbstractApi(HttpClientWrapper httpClientWrapper) {
 
     protected Path buildPath(Path base, String... components) {
         Path p = base;
-        for (String c: components) {
-            p  = p.resolve(c + "/");
+        for (String c : components) {
+            if (!"".equals(c))
```

Comment:
This does not ignore arguments with white space.

Reply:
OK, I can use `isBlank` here as well.